README.md

```diff
@@ -340,6 +342,32 @@ a modification, please feel free to create an issue on Github. _Note: The issue
 template isn't optional, so please be sure not to remove it, and please fill it
 out completely._
 
+## Other servers
+
+While Express is the primary target of this middleware, support for other servers [is in-progress](https://github.com/webpack/webpack-dev-middleware/pull/617).
```

Comment:
I don't think we need this text so I want to merge that PR at first.

Reply:
Removed!